hydra-node/json-schemas/logs.yaml

```diff
@@ -589,98 +589,135 @@ definitions:
 
   HeadState:
     oneOf:
-      - title: "IdleState"
+      - title: "Idle"
         additionalProperties: false
         required:
           - tag
-          - chainState
+          - contents
```

Comment:
Opinion: I don't like this added level of indirection. Before, all fields in the `IdleState` data structure convey some meaning, after we have a non-descript `contents` field within which one has to look for interesting fields.
Suggestion: Do not change the external (API) representation of the data structure which implies some clever (or manual) derivation of the `ToJSON/FromJSON` instances

Reply:
See https://github.com/input-output-hk/hydra/pull/725/commits/54004cc2b3b82e7edc115b3b6884cda4bc903b63. Not sure if it's worth it :/

Reply:
Fair enough.